include/xtensor/xbuilder.hpp

```diff
@@ -761,12 +761,12 @@ namespace xt
         // The following shape calculation code is an almost verbatim adaptation of numpy:
         // https://github.com/numpy/numpy/blob/2aabeafb97bea4e1bfa29d946fbf31e1104e7ae0/numpy/core/src/multiarray/item_selection.c#L1799
         auto ret_shape = make_sequence<shape_type>(dimension - 1, 0);
-        std::size_t dim_1 = shape[axis_1];
-        std::size_t dim_2 = shape[axis_2];
+        int dim_1 = static_cast<std::size_t>(shape[axis_1]);
+        int dim_2 = static_cast<std::size_t>(shape[axis_2]);
```

Comment:
weird to `static_cast` to size_t and assign to int.

Reply:
good catch, that's a mistake